Assets/Scripts/Cat.cs

```diff
@@ -12,25 +12,40 @@ public class Cat : MonoBehaviour
 	// The cat's current activity/behavior/goal
 	CatActivity activity;
 	// Tracks last time Update() was called for dt calculation
+	BehaviorTree behaviorTree;
+	Context contextObject;
+	
 	float last_update_time;
 
     // Start is called before the first frame update
     void Start()
     {
+		/*
```

Comment:
Imagine this was just for debugging. We probably don't want to disable saving functionality